src/prebid.js

```diff
@@ -171,6 +171,36 @@ function getWinningBidTargeting() {
   return winners;
 }
 
+exports.getWinningBidTargeting = function() {
+  return getWinningBidTargeting(...arguments);
+};
+
+/**
+ * Get custom targeting keys for bids that have `alwaysUseBid=true`.
+ */
+function getAlwaysUseBidTargeting() {
+  return pbjs._bidsReceived.map(bid => {
+    if (bid.alwaysUseBid) {
+      const standardKeys = CONSTANTS.TARGETING_KEYS;
+      return {
+        [bid.adUnitCode]: Object.keys(bid.adserverTargeting, key => key).map(key => {
+          
+          // Get only the non-standard keys of the losing bids, since we
+          // don't want to override the standard keys of the winning bid.
+          if (standardKeys.indexOf(key) > -1) {
```

Comment:
Ok, let's leave it as is and we'll see about adding a polyfill for `Array.prototype.includes` later.